src/extension/actions.js

```diff
@@ -80,6 +62,22 @@ async function updateAuthToken({
   return 'invalid message';
 }
 
+function notificationConfirmCallback(tabId) {
+  return async () => {
+    /* istanbul ignore next */
```

Comment:
Not sure how to get coverage on this since the we don't test e2e between the background script and the application.